test/fixtures/v3000/r_group_block.mol

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Is this used in an automated test?

Reply:
Yup, for an IO test - https://github.com/schrodinger/ketcher/blob/master/test/io/io.js, which 
- Constructs (reads or parses) the Ketcher Struct from the given test representation
- Writes back the representation using the writer
- Compares the output with the stored test representation